source/adaptive_load/scoring_function_impl.cc

```diff
@@ -0,0 +1,64 @@
+#include "adaptive_load/scoring_function_impl.h"
+
+#include <cmath>
+
+#include "envoy/registry/registry.h"
+
+#include "external/envoy/source/common/protobuf/protobuf.h"
+
+#include "api/adaptive_load/scoring_function_impl.pb.h"
+
+namespace Nighthawk {
+
+std::string BinaryScoringFunctionConfigFactory::name() const { return "nighthawk.binary"; }
```

Comment:
nighthawk.binary doesn't seem to be an descriptive enough name, we might need to add nighthawk.binary_scoring.

Reply:
Done